src/main/java/de/voidtech/gerald/commands/fun/LmgtfyCommand.java

```diff
@@ -0,0 +1,57 @@
+package main.java.de.voidtech.gerald.commands.fun;
+
+import main.java.de.voidtech.gerald.annotations.Command;
+import main.java.de.voidtech.gerald.commands.AbstractCommand;
+import main.java.de.voidtech.gerald.commands.CommandCategory;
+import net.dv8tion.jda.api.entities.Message;
+
+import java.util.List;
+
+@Command
+public class LmgtfyCommand extends AbstractCommand {
+
+    @SuppressWarnings("FieldCanBeLocal")
+    private final String googleQuery = "https://www.google.com/search?q=";
+
+    @Override
+    public void executeInternal(Message message, List<String> args) {
+        String replyURL = "Follow this: " + googleQuery + String.join("+", args);
+        message.getChannel().sendMessage(replyURL).queue();
+    }
+
+    @Override
+    public String getDescription() {
+        return "LMGTFY (Let Me Google That For You), when someone asks a very simple question that google can answer. Use this.";
+    }
+
+    @Override
+    public String getUsage() {
+        return "lmgtfy <query>";
```

Comment:
Additionally we use square brackets for arguments in help messages, nothing major but consistency is important